package.json

```diff
@@ -12,44 +12,44 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "14.3.0",
-    "@angular/common": "14.3.0",
-    "@angular/compiler": "14.3.0",
-    "@angular/core": "14.3.0",
-    "@angular/forms": "14.3.0",
-    "@angular/platform-browser": "14.3.0",
-    "@angular/platform-browser-dynamic": "14.3.0",
-    "@angular/router": "14.3.0",
-    "rxjs": "~6.5.4",
-    "tslib": "^2.0.0",
+    "@angular/animations": "15.2.10",
+    "@angular/common": "15.2.10",
+    "@angular/compiler": "15.2.10",
+    "@angular/core": "15.2.10",
+    "@angular/forms": "15.2.10",
+    "@angular/platform-browser": "15.2.10",
+    "@angular/platform-browser-dynamic": "15.2.10",
+    "@angular/router": "15.2.10",
+    "rxjs": "6.5.4",
+    "tslib": "2.6.2",
     "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "14.2.11",
-    "@angular/cli": "14.2.11",
-    "@angular/compiler-cli": "14.3.0",
+    "@angular-devkit/build-angular": "15.2.10",
+    "@angular/cli": "15.2.10",
+    "@angular/compiler-cli": "15.2.10",
     "@project-sunbird/client-services": "5.1.2",
     "@project-sunbird/common-consumption": "7.0.5",
-    "@types/jasmine": "~3.5.0",
-    "@types/jasminewd2": "~2.0.3",
+    "@types/jasmine": "3.5.0",
+    "@types/jasminewd2": "2.0.3",
     "@types/node": "14.14.31",
-    "codelyzer": "^6.0.0",
-    "jasmine-core": "~3.8.0",
-    "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~6.4.1",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "~3.0.2",
-    "karma-jasmine": "~4.0.0",
-    "karma-jasmine-html-reporter": "^1.5.0",
+    "codelyzer": "6.0.2",
+    "jasmine-core": "3.8.0",
+    "jasmine-spec-reporter": "5.0.0",
+    "karma": "6.4.1",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-coverage-istanbul-reporter": "3.0.2",
+    "karma-jasmine": "4.0.0",
+    "karma-jasmine-html-reporter": "1.5.0",
     "lodash-es": "4.17.15",
-    "ng-packagr": "^13.3.1",
-    "protractor": "~7.0.0",
-    "ts-node": "~8.3.0",
-    "tslint": "~6.1.0",
-    "typescript": "~4.6.4"
+    "ng-packagr": "13.3.1",
+    "protractor": "7.0.0",
+    "ts-node": "8.3.0",
+    "tslint": "6.1.0",
+    "typescript": "4.9.4"
   },
   "engines": {
-    "node": ">=16.19.0",
-    "npm": ">=8.19.3"
-  }
+    "node": ">=18.19.0",
```

Comment:
why u have added 18

Reply:
it works with " >=16.19.0 <=18.19.0",i will change this